src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/ConfirmedAppointmentDetailsPage.jsx

```diff
@@ -53,6 +53,8 @@ function formatHeader(appointment) {
     return 'VA Video Connect at an ATLAS location';
   } else if (isVideoHome(appointment)) {
     return 'VA Video Connect at home';
+  } else if (isVAPhoneAppointment(appointment)) {
+    return 'VA Appointment over the phone';
```

Comment:
FYI, Just did a PR where some these helper selectors (isVideoHome, isVAPhoneAppointment, etc.) where removed due to them not adding much value. The alternative is to get the information directly from the appointment object.

Reply:
Yeah, I think this helper is a good candidate for removal, but we may want to leave that to a separate ticket so we can remove all the uses of it at once.